src/test/java/ru/spbstu/calculator/SimilarityScoreCalculatorTest.java

```diff
@@ -10,45 +10,82 @@
 
 public class SimilarityScoreCalculatorTest {
 
-    private FastaFile fastaFile;
+    private final String fastaPW = "src/test/resources/PWTest";
+    private final String fastaCalculate = "src/test/resources/calculator-test";
     private final double delta = 0.0001;
+    private final int sequenceLength = 9;
+
+    private DatasetScore ds;
 
     @Before
     public void setup() throws IOException {
-        fastaFile = new FastaFile();
-        final String fileName = "src/test/resources/calculator-test";
-        fastaFile.readFastaFile(fileName);
+        FastaFile fastaFile = new FastaFile();
+        fastaFile.readFastaFile(fastaCalculate);
+
+        FastaFile fastaFilePW = new FastaFile();
+        fastaFilePW.readFastaFile(fastaPW);
+        PWCalculator pwCalculator = new PWCalculator();
+        PWMatrix matrix = pwCalculator.calculateMatrix(fastaFilePW, CalculationStrategy.builder()
+                .sequenceLength(sequenceLength)
+                .resultFrequency(Constants.calculationMethod.RELATIVE)
+                .build());
+
+        SimilarityScoreCalculator ssc = new SimilarityScoreCalculator(sequenceLength, fastaFile);
+        ds = ssc.calculateScore(matrix);
```

Comment:
better to extract the duplicated code in another method or even class (maybe abstract). It helps to maintain tests easier when the base code is changed.